torch/_inductor/codegen/cpp.py

```diff
@@ -3653,30 +3653,36 @@ def _can_fuse_nodes_with_compatible_ranges(self, node1, node2):
         if isinstance(node_to_recomp, FusedSchedulerNode):
             return False
```

Comment:
Not for this PR, but w should probably also allow to fuse Fused nodes here, right?
I remember we discussed it in the other PR, but I don't remember what we resolvd.